ui/mirage/scenarios/sync.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+export default function (server) {
+  const types = ['aws-sm', 'azure-kv', 'gcp-sm', 'gh', 'vercel-project'];
```

Comment:
Once this is merged, I can update this to use the helper I made!

Reply:
Nice ya I was going to leave a comment about that but you didn't need it 😁